instrumentation/go.mongodb.org/mongo-driver/v2/mongo/otelmongo/example_test.go

```diff
@@ -45,3 +62,39 @@ func Example() {
 		panic(err)
 	}
 }
+
+func ExampleWithSpanNameFormatter() {
+	// connect to MongoDB
+	opts := options.Client()
+	opts.Monitor = otelmongo.NewMonitor(
+		otelmongo.WithSpanNameFormatter(func(event *event.CommandStartedEvent) string {
+			collection := extractCollection(event)
```

Comment:
I'm fine with a comment explaining how one can retrieve the collection name. But duplicating the method for the sake of an example seems a bit too much.
The example doesn't need to use the collection name right?